src/contract_wrappers/tokens/__tests__/erc20_wrapper.test.ts

```diff
@@ -0,0 +1,353 @@
+
+      // Real call
+      const result = await target.decimals();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.decimals).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('balanceOf', () => {
+    test.todo('should fail as owner is not an Eth address');
+    test('should call to balanceOf', async () => {
+      const expectedResult = new BigNumber(0);
+      const params = {
+        owner: '0x1111111111111111111111111111111111111111',
+      };
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.balanceOf).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync(params.owner)).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.balanceOf(params);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.balanceOf).once();
+      verify(mockedMethod.callAsync(params.owner)).once();
+    });
+  });
+
+  describe('symbol', () => {
+    test('should call to symbol', async () => {
+      const expectedResult = 'string';
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.symbol).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync()).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.symbol();
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.symbol).once();
+      verify(mockedMethod.callAsync()).once();
+    });
+  });
+
+  describe('allowance', () => {
+    test.todo('should fail as owner is not an Eth address');
+    test.todo('should fail as spender is not an Eth address');
+    test('should call to allowance', async () => {
+      const expectedResult = new BigNumber(0);
+      const params = {
+        owner: '0x1111111111111111111111111111111111111111',
+        spender: '0x2222222222222222222222222222222222222222',
+      };
+      // Mocked method
+      const mockedMethod = mock(MockedCallMethod);
+      // Stub the method
+      when(mockedContract.allowance).thenReturn(instance(mockedMethod));
+      // Stub the request
+      when(mockedMethod.callAsync(params.owner, params.spender)).thenResolve(expectedResult);
+
+      // Real call
+      const result = await target.allowance(params);
+      // Result expectation
+      expect(result).toBe(expectedResult);
+      // Verifications
+      verify(mockedContract.allowance).once();
+      verify(mockedMethod.callAsync(params.owner, params.spender)).once();
+    });
+  });
+
+  describe('isValidContract', () => {
+    test('should call to isValidContract', async () => {
```

Comment:
You aren't calling the real method haha.
I realize it when I checked the coverage report. 
Add the call and check if the test is passing.

Reply:
I called each module individually

Reply:
It is not ok. You are testing `isValidContract` so you need to make a real call to that method here. 
For each of those individual methods must be a different tests. 
This way, we are not covering `isValidContract` method, so if anyone makes a change or delete it, tests won't fail and we won't notice it.